module/helpers.nix

```diff
@@ -1,5 +1,8 @@
 { lib }:
-{
+rec {
+  secretFilesRoot = "/tmp/detsys-vault/";
+  environmentFilesRoot = "/run/keys/environment/";
```

Comment:
Probably best to make this variable per service by including the service name in this variable, as opposed to updating all the uses of this variable.